doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -229,6 +229,45 @@ The following table indicates the software maturity levels of the support for ea
 
   .. sml-table:: wifi
 
+Ecosystem support
+*****************
+
+The following sections contain the tables indicating the software maturity levels of the support for the following ecosystems:
+
+* Google Fast Pair
+
+Google Fast Pair
+================
+
+The following table indicates the software maturity levels of the support for each Google Fast Pair use case:
```

Comment:
`for Google Fast Pair use cases integrated in the NCS` (e.g. we do not integrate audio use-cases at all)